wp-parsely.php

```diff
@@ -1,11 +1,10 @@
 <?php
 /**
- * Parse.ly
+ * Parse.ly plugin loader
  *
- * @package      Parsely\wp-parsely
- * @author       Parse.ly
- * @copyright    2012 Parse.ly
- * @license      GPL-2.0-or-later
+ * Requires any necessary files and loads the plugin.
+ *
+ * @package Parsely
```

Comment:
Would this modifications have some effects on how WordPress.org represents our plugin?

Reply:
Theoretically no, (although I suspect line 3 could be a potential problem) as WP looks at [header fields](https://developer.wordpress.org/plugins/plugin-basics/header-requirements/) and Packagist uses the [composer.json file](https://packagist.org/#how-to-submit-packages).

However, I've restored this just to be sure we're safe.